protocol/activity_center_persistence.go

```diff
@@ -816,27 +816,43 @@ func (db sqlitePersistence) MarkActivityCenterNotificationsUnread(ids []types.He
 
 }
 
-func buildActivityCenterNotificationsCountQuery(isAccepted bool) string {
+func (db sqlitePersistence) buildActivityCenterNotificationsCountQuery(isAccepted bool, activityTypes []ActivityCenterType) *sql.Row {
+	var args []interface{}
 	var acceptedWhere string
 
 	if !isAccepted {
 		acceptedWhere = `AND NOT accepted`
 	}
 
-	return fmt.Sprintf(`SELECT COUNT(1) FROM activity_center_notifications WHERE NOT read AND NOT dismissed %s`, acceptedWhere)
+	var inTypeWhere string
```

Comment:
Since generating the count query is now more involved, it would be better if we had tests for the function `UnreadAndAcceptedActivityCenterNotificationsCount`, for example in `persistence_test.go`.

Reply:
Done!